gooddata-java-model/src/test/groovy/com/gooddata/sdk/model/md/visualization/VisualizationObjectTest.groovy

```diff
@@ -154,10 +164,10 @@ class VisualizationObjectTest extends Specification {
 
         where:
         measures                                    | expected
-        multipleMeasuresVisualization.getMeasures() | [ multipleMeasuresVisualization.buckets[0].items[0],
-                                                        multipleMeasuresVisualization.buckets[1].items[0],
-                                                        multipleMeasuresVisualization.buckets[1].items[1] ]
-        emptyBucketsVisualization.getMeasures()       | []
+        multipleMeasuresVisualization.getMeasures() | [multipleMeasuresVisualization.buckets[0].items[0],
+                multipleMeasuresVisualization.buckets[1].items[0],
```

Comment:
Original indentation looked more readable to me here. It should be possible to use
```
// @formatter:off
...
// @formatter:on
```

Reply:
done